extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -107,7 +107,9 @@ function tests(name, installer) {
           type: 'a9',
           src: 'testsrc',
           resizable: '',
-        }, 'https://schema.org').then(element => {
+        }, 'https://schema.org', undefined, () => {
+          clock = sandbox.useFakeTimers();
```

Comment:
Can you explain this change. It might not be the right thing to do, but it looks like it was painful to find this :)